cmake/external/onnxruntime_external_deps.cmake

```diff
@@ -176,10 +176,11 @@ set(ENABLE_DATE_TESTING  OFF CACHE BOOL "" FORCE)
 set(USE_SYSTEM_TZ_DB  ON CACHE BOOL "" FORCE)
 
 FetchContent_Declare(
-      date
-      URL ${DEP_URL_date}
-      URL_HASH SHA1=${DEP_SHA1_date}
-    )
+  date
+  URL ${DEP_URL_date}
+  URL_HASH SHA1=${DEP_SHA1_date}
+  FIND_PACKAGE_ARGS NAMES date
```

Comment:
For reviewers, this is only required if `FETCHCONTENT_TRY_FIND_PACKAGE_MODE == OPT_IN`.

Reply:
Nevermind, it's good to have in case OPT_IN is used.